bounties_api/std_bounties/models.py

```diff
@@ -28,6 +28,17 @@ def save(self, *args, **kwargs):
         self.normalized_name = self.name.lower().strip()
         super(Category, self).save(*args, **kwargs)
 
+class Language(models.Model):
+    _6391 = models.CharField(max_length=2)
```

Comment:
Thinking we don't need these for now 639-1, 639-2/T, 639-2/B

Reply:
👍